android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotification.java

```diff
@@ -68,6 +68,11 @@ public void onNewIntent(Intent intent) {
             bundle.putBoolean("foreground", false);
             intent.putExtra("notification", bundle);
             notifyNotification(bundle);
+        }else if ( intent.getExtras() != null ) {
+            Bundle bundle = intent.getExtras();
+            bundle.putBoolean("foreground", false);
+            intent.putExtra("notification", bundle);
+            notifyNotification(bundle);
         }
     }
```

Comment:
Can you please tell me how to test this, I'm unable to create a notification that when I tap it in the notification centre it hits this new 'else' block.